File: Java/Card.java
class Card extends Payment {
  Integer number;
  Integer cvv;
  String date;

  public Card(Integer id, Integer number, Integer cvv, String date) {
    super(id);
    this.number = number;
    this.cvv = cvv;
    this.date = date;
  }

  void Datos() {
    System.out.println("number: " + number + "\n" +
    "cvv: " + cvv + "\n" + "date: " + date);
  }

}

File: PHP/Card.php
<?php
require_once('Payment.php');

class Card extends Payment {
  public $number;
  public $cvv;
  public $date;

  public function __construct($id, $number, $cvv, $date) {
    parent::__construct($id);
    $this->number = $number;
    $this->cvv = $cvv;
    $this->date = $date;
  }

  public function Datos() {
    echo " NUMBER:".$this->number ." CVV:".$this->cvv. " Date:".$this->date;
  }
}